apps/api/.env.example

```
@@ -32,7 +32,11 @@ EMAIL_VERIFICATION_TTL_SECONDS=600
 EMAIL_VERIFICATION_RESEND_COOLDOWN_SECONDS=60
 EMAIL_VERIFICATION_MAX_ATTEMPTS=5
 
-# Resend email delivery
-RESEND_API_KEY="replace_with_resend_api_key"
-RESEND_FROM_EMAIL="noreply@ai-study-hub.local"
-RESEND_FROM_NAME="AI Study Hub"
+# Mailtrap email delivery
+MAILTRAP_SMTP_HOST="sandbox.smtp.mailtrap.io"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)